springwolf-core/src/main/java/io/github/stavshamir/springwolf/asyncapi/types/channel/operation/message/PayloadReference.java

```diff
@@ -3,12 +3,16 @@
 import lombok.EqualsAndHashCode;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
+import lombok.ToString;
 
 @NoArgsConstructor
 @EqualsAndHashCode
+@ToString
 public class PayloadReference {
 
     @Getter
+    @EqualsAndHashCode.Include
+    @ToString.Include
     private String $ref;
```

Comment:
FYI: lombok EqualsAndHashCode and ToString ignore fields with a `$` prefix.
Following, also some test a fixed.

Reply:
Good to know, thanks